src/actions/ast.js

```diff
@@ -105,7 +109,7 @@ function compressPausePoints(pausePoints) {
     compressed[line] = {};
     for (const col in pausePoints[line]) {
       const point = pausePoints[line][col];
-      compressed[line][col] = (point.break && 1) | (point.step && 2);
+      compressed[line][col] = (point.break ? 1 : 0) | (point.step ? 2 : 0);
```

Comment:
haha - but this was the most clever code I ever wrote :)

Reply:
It is clever; too clever for flow it seems. :)

Reply:
haha aww shucks